src/components/DetailView/vegaSpec.js

```diff
@@ -118,6 +118,13 @@ export function colorEncoding(selections) {
  * @param {Array<string>} title
  */
 export function createSpec(sensor, primaryValue, selections, initialSelection, title) {
+  const ratioSuffix =
```

Comment:
an alternative strategy is using the `$signalCasesOrDeathOptions` (with `.ratio` and `.cumulative`) as way to differentiate variants

Reply:
That would be cleaner, but it's not working for me.

```
  const ratioSuffix = $signalCasesOrDeathOptions && $signalCasesOrDeathOptions.ratio ? ' per 100,000 people' : '';
  const yAxisTitle = sensor.yAxis + ratioSuffix;

  const leftPadding = $signalCasesOrDeathOptions && $signalCasesOrDeathOptions.cumulative ? 90 : 50;
```